NitroxClient/GameLogic/InitialSync/EquippedItemInitialSyncProcessor.cs

```diff
@@ -32,44 +35,37 @@ public override IEnumerator Process(InitialPlayerSync packet, WaitScreen.ManualW
 
         using (PacketSuppressor<EntitySpawnedByClient>.Suppress())
         {
-            foreach (EquippedItemData equippedItem in packet.EquippedItems)
+            foreach (var pair in packet.EquippedItems)
```

Comment:
We don't use var cause it's harder to directly see what's the type of the pair in question even if we have its decomposition below.